model/event/subscriptions.go

```diff
@@ -33,6 +34,13 @@ var (
 	subscriberWithRegexRegexSelectorsKey = bsonutil.MustHaveTag(subscriberWithRegex{}, "RegexSelectors")
 )
 
+type OwnerType string
```

Comment:
does it make sense to put these constants in the evergreen package with the other globals?

Reply:
I think it makes more sense here, since it's only used for notifications and their rest routes